config/sudoers-for-dev

```diff
@@ -25,6 +25,8 @@ vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/set-clock.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/lockdown.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/generate-key.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/setup-boot-entry.sh
+vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/create-machine-cert.sh
+vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/program-system-administrator-cards.sh
```

Comment:
Missed this `sudoers-for-dev` file when I updated `sudoers` in https://github.com/votingworks/vxsuite-complete-system/pull/291